src/main/java/ticket/TicketService.java

```diff
@@ -47,5 +49,25 @@ public static void main(String[] args) {
         }
         users.delete(client);
         System.out.println(users.contain(client));
+
+        TicketDao ticketDao = new TicketDao();
+        UserDao userDao = new UserDao();
+        Ticket ticket1 = new Ticket();
+        ticket1.setId(1);
```

Comment:
why? isn't his id already set generated and set by default?

Reply:
Fix in https://github.com/kolosokvit/ticket-service/pull/4/commits/212acc623c5a6a20a5cd69def1f49da4e8b8d9c3